project.godot

```diff
@@ -26,7 +26,8 @@ Packet="*res://scripts/multiplayer/packet.gd"
 Anticheat="*res://scripts/multiplayer/anticheat.gd"
 Settings="*res://scripts/settings.gd"
 Modules="*res://scripts/modules.gd"
-TauntModule="res://modules/taunt/taunt.gd"
+KeywordModule="*res://modules/keyword/keyword.gd"
```

Comment:
`KeywordModule` and all base modules need to be global variables to be used by child modules. There should be something to discourage referencing modules directly somewhere for other users, but it's fine for now.